tests/README.md

```diff
@@ -26,7 +25,8 @@ sockets. It works with both a real Nano S/X wallet, or Speculos. As the CI of
 this repository does not have access to a Nano S/X, it uses Speculos.
 
 The functional tests using Speculos (`tests/speculos/` directory) or LedgerComm
-+ Speculos (`tests/ledgercomm/` directory) are the same, use the same language
```

Comment:
I think the correction is incorrect here. The expected sentence is
> The functional tests using Speculos (`tests/speculos/` directory) or LedgerComm + Speculos (`tests/ledgercomm/` directory) are the same, use the same language

Maybe replacing the "+" with an "&" or "and" would fix the misinterpretation.

Reply:
Replaced it with `\+`